renderdoc/driver/d3d12/d3d12_common.h

```diff
@@ -380,6 +384,7 @@ void Serialiser::Serialise(const char *name, D3D12Descriptor &el);
   D3D12_CHUNK_MACRO(CREATE_HEAP, "ID3D12Device::CreateHeap")                                       \
   D3D12_CHUNK_MACRO(CREATE_COMMITTED_RESOURCE, "ID3D12Device::CreateCommittedResource")            \
   D3D12_CHUNK_MACRO(CREATE_PLACED_RESOURCE, "ID3D12Device::CreatePlacedResource")                  \
+  D3D12_CHUNK_MACRO(CREATE_RESERVED_RESOURCE, "ID3D12Device::CreateReservedResource")              \
```

Comment:
These should probably go on the end of the list now. The enums defined here are serialised in the capture - adding these new chunks into the middle will offset the values of the other chunks. So when replaying a capture made before these were added the chunks will be all wrong and start reading garbage data.

We could bump the serialise version and make those captures incompatible, but I think it's better just to slightly mess up the organisation and add them at the end.